### ISSUE TITLE
panel is not brought to front when clicking on header

### ISSUE THREAD

**Issue by yarekc** [Flyer53/jsPanel3#45]
I don't know if this is a real bug, however it does not react "naturally": 

When clicking on the header bar of the panel, panel is not brought to front: 
http://jspanel.de/api/#gettingstarted

It is brought to front when clicking in his body part

Regards

**Comment by Flyer53** [comment]
Hey there,
I thought I fixed this ... it does work on FF. Will have to look at this again.
Meanwhile you could try this workaround:
```javascript
$.jsPanel({
    callback: function(panel){
        $(panel.header.title).on('mousedown', function(){
            panel.front();
        });
    }
});
```
Thanks for reporting 😄 
Regards, Stefan

**Comment by Flyer53** [comment]
OK, I have a preliminary v3.9.3 that finally seems to fix this.
I need to do some more testing, so it's not an official release yet. But you can download and test it.
http://jspanel.de/downloads/jsPanel-3.9.3.zip
Feedback would be very much appriciated :)

**Comment by Flyer53** [comment]
**fixed with release v3.9.3**
